runtime/bcutil/cfreader.c

```diff
@@ -2575,22 +2575,22 @@ checkClassVersion(J9CfrClassFile* classfile, U_8* segment, U_32 vmVersionShifted
 	const U_16 max_allowed_version = vmVersionShifted >> BCT_MajorClassFileVersionMaskShift;
```

Comment:
Suggest this would be better named `maxMajorVersion` for consistency with other local variables and its use.